### PR TITLE
unset_default_float_dtype #6647  - added doc string examples

### PR DESCRIPTION
**Pull request by ojumah20**
add_reformatting_checklist_DataType

### CODE REVIEW

**Comment by ojumah20** [comment]
add_reformatting_checklist_DataType

File: ivy/functional/ivy/data_type.py [hunk 1]
```diff
@@ -1861,7 +1861,27 @@ def unset_default_dtype():
...
+    """Reset the current default float dtype to the previous state.
+
+    Examples
+    --------
+    >>> ivy.set_default_float_dtype(ivy.float64)
+    >>> ivy.default_float_dtype()
+    'float64'
+    >>> ivy.unset_default_float_dtype()
+    >>> ivy.default_float_dtype()
+    'float32'
```
Comment (AnnaTz) [inline ivy/functional/ivy/data_type.py:1873]: I don't think this example is useful. You can remove it and keep only the other one.

File: ivy/functional/ivy/data_type.py [hunk 2]
```diff
@@ -1861,7 +1861,27 @@ def unset_default_dtype():
...
+
+
+    >>> ivy.set_default_float_dtype(ivy.float32)
+    >>> ivy.set_default_float_dtype(ivy.float64)
+    >>> ivy.default_float_dtype_stack
+    ['float32','float64']
+
```
Comment (AnnaTz) [inline ivy/functional/ivy/data_type.py:1880]: No need for this blank line here.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

File: ivy/functional/ivy/data_type.py (hunk 1 above)
Comment (ojumah20) [reply ivy/functional/ivy/data_type.py]: alright, I have removed it

File: ivy/functional/ivy/data_type.py (hunk 2 above)
Comment (ojumah20) [reply ivy/functional/ivy/data_type.py]: Okay, i've removed it

**Review by AnnaTz — APPROVED** [review]
Looks good to me! Ready to be merged 🙂

**Comment by ojumah20** [comment]
Thank you

**Pull request merged**